chart/flux/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 appVersion: "1.19.0"
-version: 1.3.0
+version: 1.3.1
```

Comment:
We bump the chart version during the release.

Reply:
Reverted